keyboards/hecomi/hecomi.h

```diff
@@ -0,0 +1,115 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef HECOMI_H
+#define HECOMI_H
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+//7,7,6,6,6
+#define LAYOUT_LEFT(\
+K00, K01, K02, K03, K04, K05, K06, \
+K10, K11, K12, K13, K14, K15, K16, \
+K20, K21, K22, K23, K24, K25,  \
+K30, K31, K32, K33, K34, K35,  \
+K40, K41, K42, K43, K44, K45 \
+)  {\
+{K00, K00, K01, K02, K03, K04, K05, K06, KC_NO},\
+{K10, K10, K11, K12, K13, K14, K15, K16, KC_NO},\
+{K20, K20, K21, K22, K23, K24, K25, KC_NO, KC_NO},\
+{K30, K30, K31, K32, K33, K34, K35, KC_NO, KC_NO},\
+{K40, K40, K41, K42, K43, K44, K45, KC_NO, KC_NO}\
+}
+
+#define LAYOUT_RIGHT(\
+K00, K01, K02, K03, K04, K05, K06, K07, \
+K10, K11, K12, K13, K14, K15, K16, K17, \
+K20, K21, K22, K23, K24, K25, K26, K27, \
+K30, K31, K32, K33, K34, K35, K36, K37, \
+K40, K41, K42, K43, K44, K45, K46, K47\
+) {\
+{K00, K00, K01, K02, K03, K04, K05, K06, K07},\
+{K10, K10, K11, K12, K13, K14, K15, K16, K17},\
+{K20, K20, K21, K22, K23, K24, K25, K26, K27},\
+{K30, K30, K31, K32, K33, K34, K35, K36, K37},\
+{K40, K40, K41, K42, K43, K44, K45, K46, K47}\
+}
+
+/*
+7
+7
+6
+6
+6
+
+32
+
+8
+8
+7
+8
+6
+
+37
+*/
+
+#define LAYOUT(\
+	K00, K01, K02, K03, K04, K05, K06,        K50, K51, K52, K53, K54, K55, K56, K57, \
+	K10,   K11, K12, K13, K14, K15, K16,    K60, K61, K62, K63, K64, K65, K66,   K67, \
+	K20,    K21, K22, K23, K24, K25,          K71, K72, K73, K74, K75, K76,      K77,\
+	K30,     K31, K32, K33, K34, K35,      K80, K81, K82, K83, K84, K85, K86,    K87, \
+	K40,  K41, K42, K43, K44, K45,        K92, K93, K94, K95, K96, K97\
+) {\
+	{K00, K01, K02, K03, K04, K05, K06, KC_NO},\
+	{K10, K11, K12, K13, K14, K15, K16, KC_NO},\
+	{K20, K21, K22, K23, K24, K25, KC_NO,    KC_NO},\
+	{K30, K31, K32, K33, K34, K35, KC_NO,    KC_NO},\
+	{K40, K41, K42, K43, K44, K45, KC_NO,    KC_NO},\
+	{K50, K51, K52, K53, K54, K55, K56, K57},\
+	{K60, K61, K62, K63, K64, K65, K66, K67},\
+	{KC_NO,    K71, K72, K73, K74, K75, K76, K77},\
+	{K80, K81, K82, K83, K84, K85, K86, K87},\
+	{KC_NO,    KC_NO,    K92, K93, K94, K95, K96, K97}\
+}
+
+#define LAYOUT_KC(\
```

Comment:
Honestly, we try to avoid this layout macro. 

But if you want to use it, it should be:
```suggestion
#define LAYOUT_kc(\
```